src/spiritual/spiritual-mix/gui-keys@wunderbyte.com/module.js

```diff
@@ -129,8 +129,21 @@ gui.KeysModule = gui.module('gui-keys@wunderbyte.com', {
 	 * @param {String} sig Contextkey
 	 */
 	_maybebroadcast: function(down, key, c, code, sig) {
-		if (!c) {
-			this._broadcast(down, key, c, code, sig);
+		switch (code) {
```

Comment:
might as well add all the codes from https://w3c.github.io/uievents/#fixed-virtual-key-codes

did a quick regex on the table: `(.*)\t(\d+)` => `case $2: // $1`

```
case 8: // Backspace	
case 9: // Tab	
case 13: // Enter	
case 16: // Shift	
case 17: // Control	
case 18: // Alt	
case 20: // CapsLock	
case 27: // Escape
case 32: // Space	
case 33: // PageUp	
case 34: // PageDown	
case 35: // End	
case 36: // Home	
case 37: // ArrowLeft	
case 38: // ArrowUp	
case 39: // ArrowRight	
case 40: // ArrowDown	
case 46: // Delete
```

Reply:
👍